src_bonus/parse/parse_to_num_bonus.c

```diff
@@ -27,20 +31,18 @@ struct s_vec3	vec_get(char *s, double min, double max)
 	struct s_vec3	vec;
 	char			*s_tmp;
 	char			*s_tmp_ptr;
-	int				i;
 
-	i = 0;
 	s_tmp = ft_strdup(s);
 	s_tmp_ptr = s_tmp;
 	vec.x = double_get(ft_strsep(&s_tmp, ','), min, max);
 	vec.y = double_get(ft_strsep(&s_tmp, ','), min, max);
 	vec.z = double_get(ft_strsep(&s_tmp, ','), min, max);
 	if (ft_strsep(&s_tmp, ',') != NULL)
-		error_user("Elements must came in standard.\n");
+		error_user("[x,y,z] elements must came in standard.\n");
```

Comment:
이건 rgb에서도 걸릴 문제 같아서...ㅎ
```suggestion
		error_user("[x,y,z] or [r,g,b] elements must came in standard.\n");
```

Reply:
이런 디테일한 부분 좋아요